src/style.css

```diff
@@ -0,0 +1,142 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+}
+
+body,
+.card__tittle,
+.card__code,
+.card__reviews {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto;
+}
+
+.card__tittle {
+  margin-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.card__code {
+  margin-top: 4px;
+  display: block;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+
+}
+
+.card__review-container {
+  margin-top: 16px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__reviews {
+  color: #060b35;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+.card__price-container {
+  margin-top: 24px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.card__price-label {
+  color: #616070;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.card__price {
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  color: #060b35;
+}
+
+.card__buy-button {
+  margin-top: 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 40px;
+  background-color: #00acdc;
+  border-radius: 5px;
+  border: 1px solid transparent;
+
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  text-transform: uppercase;
+  text-decoration: none;
+  color: white;
+
+  transition: color 0.3s, background-color 0.3s;
+}
+
+.card__buy-button:hover {
+  background-color: #fff;
+  border-color: #00acdc;
+  color: #00acdc;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url("images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n+1) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url("images/star-active.svg");
+}
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url("images/star-active.svg");
+}
```

Comment:
CHECKLIST RULE 11 [FILE STRUCTURE] - You should create separate file for each BEM block,.

Reply:
About variables and colors. I think in this level it's not important, because there is no concretic elements and colors to unit and manage them. But it interesting for the feature. Thanks.

Reply:
You have this rule in checklist, so it's important, implement it

Reply:
let's fix this comment